sig-contributor-experience/meeting-notes-archive/2018-meeting-notes.md

```diff
@@ -2851,4 +2851,4 @@ Agenda:
     *   Better Triage and Visibility into K8s Issue situation [sahdev]
     *   Documentation
     *   Removing the need for write access 
-        *   Adding milestone is the only thing missing, right? [dankohn]
\ No newline at end of file
+        *   Adding milestone is the only thing missing, right? [dankohn]
```

Comment:
this file does not need to be changed as its an archive so we should keep the history